epub33/rs/index.html

```diff
@@ -1596,25 +1599,27 @@ <h4>Basic Playback</h4>
 				<section id="sec-rsconf-timing-synch">
 					<h3>Timing and Synchronization</h3>
 
-					<p id="mol-timing-sync" data-tests="#mol-timing-synchronization">Reading Systems MUST render immediate children of the <a data-cite="epub-33#elemdef-smil-body"
-								><code>body</code> element</a> [[EPUB-33]] in a sequence. A <a
-							data-cite="epub-33#elemdef-smil-seq"><code>seq</code> element's</a> [[EPUB-33]] children
-						MUST be rendered in sequence, and playback completes when the last child finishes playing.
-						Reading System MUST render a <a data-cite="epub-33#elemdef-smil-par"><code>par</code>
-							element's</a> [[EPUB-33]] children in parallel (with each starting at the same time), and
-						playback completes when all the children finish playing. Reading System playback of the Media
-						Overlay Document completes when the <code>body</code> element's last child finishes playing.</p>
+					<p id="mol-timing-sync" data-tests="#mol-timing-synchronization">Reading Systems MUST render
+						immediate children of the <a data-cite="epub-33#elemdef-smil-body"><code>body</code> element</a>
+						[[EPUB-33]] in a sequence. A <a data-cite="epub-33#elemdef-smil-seq"><code>seq</code>
+							element's</a> [[EPUB-33]] children MUST be rendered in sequence, and playback completes when
+						the last child finishes playing. Reading System MUST render a <a
+							data-cite="epub-33#elemdef-smil-par"><code>par</code> element's</a> [[EPUB-33]] children in
+						parallel (with each starting at the same time), and playback completes when all the children
+						finish playing. Reading System playback of the Media Overlay Document completes when the
+							<code>body</code> element's last child finishes playing.</p>
 				</section>
 
 				<section id="sec-rsconf-rendering-audio">
 					<h5>Rendering Audio</h5>
 
-					<p id="mol-rendering-audio" data-tests="#mol-timing-synchronization">When presented with a Media Overlay <a data-cite="epub-33#elemdef-smil-audio"><code>audio</code>
```

Comment:
This is the wrong test for this section, I think it's called `#mol-rendering-audio` (?)

Reply:
We will have to check the cross references for the tests, but I would not mix this issue with this PR.

Reply:
B.t.w, I have just cleaned up this in a separate PR in #1990.

I mark this issue as resolved (for the purpose of this PR).